apps/src/tutorialExplorer/tutorialSet.jsx

```diff
@@ -17,13 +18,21 @@ const TutorialSet = React.createClass({
         className="col-80"
         style={{float: 'left'}}
       >
-        {this.props.tutorials.map(item => (
-          <Tutorial
-            item={item}
-            filters={this.props.filters}
-            key={item.code}
-          />
-        ))}
+        <ReactCSSTransitionGroup
+          transitionName="fadable"
+          transitionAppear={true}
+          transitionLeave={false}
+          transitionEnterTimeout={200}
+          transitionAppearTimeout={200}
+        >
+          {this.props.tutorials.map((item, index) => (
+            <Tutorial
+              item={item}
+              filters={this.props.filters}
+              key={`item_${item.code}_${index}`}
```

Comment:
fwiw, i dont think there's any need for these keys to be human readable - just unique. i.e. there's probably no reason to prepend with `item_`